playbooks/utils/replace_vm_host.yml

```diff
@@ -0,0 +1,75 @@
+---
+#  This playbook will delete a VM and replace it. This is the only warning you will get. Running this playbook has destructive consequences
```

Comment:
Can we put more warnings in Ansible Tower? Maybe even one of those annoying things where you have to type something into a box in order for it to really run?